sklearn_numba_dpex/patches/tests/test_patches.py

```diff
@@ -86,3 +87,82 @@ def test_spirv_fix():
                 kmeans.fit(X_array)
         finally:
             _load_numba_dpex_with_patches()
+
+
+def test_hack_906():
```

Comment:
Let's use an explicit function name:

```suggestion
def test_need_to_workaround_numba_dpex_906():
```